server/src/test/java/org/elasticsearch/discovery/HandshakingTransportAddressConnectorTests.java

```diff
@@ -166,7 +167,13 @@ public void testLogsFullConnectionFailureAfterSuccessfulHandshake() throws Excep
                 "message",
                 HandshakingTransportAddressConnector.class.getCanonicalName(),
                 Level.WARN,
-                "*completed handshake with [*] but followup connection failed*"
+                "*completed handshake with ["
```

Comment:
Is this matching for a log message or entire log line (with timestamp/thread/host/etc)?
If this is just a message do we need a `*`? Looks like this is an exact match.